tests/test_options.py

```diff
@@ -387,6 +387,36 @@ def test_opt_table_font_raises():
     assert "Either `font=` or `stack=` must be provided." in exc_info.value.args[0]
 
 
+def test_opt_row_striping():
+
+    gt_tbl_0 = GT(exibble)
+    gt_tbl_1 = GT(exibble).opt_row_striping()
+    gt_tbl_2 = GT(exibble).opt_row_striping().opt_row_striping(row_striping=False)
+
+    assert gt_tbl_0._options.row_striping_include_table_body.value == False
+    assert gt_tbl_1._options.row_striping_include_table_body.value == True
+    assert gt_tbl_2._options.row_striping_include_table_body.value == False
+
+
+def test_tab_options_striping():
+
+    gt_tbl_opt = GT(exibble, id="test").opt_row_striping()
+    gt_tbl_tab_options = GT(exibble, id="test").tab_options(row_striping_include_table_body=True)
+
+    assert gt_tbl_opt.as_raw_html() == gt_tbl_tab_options.as_raw_html()
+
+
+def test_tab_options_striping_snap(snapshot):
```

Comment:
See comment on snapshot. It'd be okay to test the css and html separately if it makes it easier to strip down. I wonder if you could easily test the css using an assertion, and snapshot test just the html body (with fewer rows of data; the minimum needed to test--3? or w/e makes sense given there are groupings)

Reply:
I'm going to make the input df with much smaller dimensions and only snapshot the table body (everything inside `<tbody>`). I don't think there's a need to snapshot the CSS part.